core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -412,11 +414,12 @@ class BrokerServer(
         replicaManager,
         time,
         shareFetchSessionCache,
-        config.shareGroupConfig.shareGroupRecordLockDurationMs,
```

Comment:
I don't quite see the point of this change. The purpose of the parameter has changed - it was the lock duration, but now its the default lock duration if the group does not override it.